integration_tests/src/main/python/parquet_pyarrow_test.py

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import difflib
+import pyarrow as pa
+import pyarrow.parquet as pa_pq
+import pytest
+import sys
+import time
+
+from asserts import _prep_incompat_conf, _RowCmp, _assert_equal
```

Comment:
I am not super happy about pulling in "private" methods. If we want to expose them, we should do it properly and rename them.

Reply:
Done.